index.html

```diff
@@ -19,7 +19,7 @@
                 &lt;script src='stream.min.js'&gt;&lt;/script&gt;
             </pre>
 
-            <a class='download' href="stream.min.js">Download stream.js</a>
+            <a class='download' href="stream.js.zip">Download stream.js</a>
```

Comment:
Because I would like to also give users the source map file and since the isn't a UI to download that, I though why not just bundle the two in a zip file. But I guess two buttons would be better then?